### ISSUE TITLE
[HW Accel Support]: Bullseye RASPI 64bit v0.11-rc(x)

### ISSUE THREAD

**Issue by S-Przybylski** [blakeblackshear/frigate#3868]
### Describe the problem you are having

Hi Blake,
I reinstalled my existing and running RASPI4 64bit buster (used frigate v0.11-rc2 docker container) installation with bullseye 64bit with several no name cam's and 2 dahua's. I reinstalled all required things, changed the gpu size to 192 and started frigate:
Now only the two dahua cam's are working (cam7,8) using hardware acceleration. All others are green. I tried the current version frigate v0.11-rc3 - same here. I have executed the ffmpeg test string you provided me last time: Now it seems that the ffmpeg stuck after several seconds (only ctrl-c 3 times helps) - differently to my old case with buster.

The config is identical without changing anything like documented in ealier issue, which worked for weeks now with buster. 
I have updated the firmware to the lasted version 5.15.68-v8+ #1586 SMP PREEMPT Fri Sep 16 12:13:50 BST 2022 aarch64 GNU/Linux - also no success. I also tried to install a 5.15.56 version - also no success. Now the 5.15.68 is installed again.

If I execute the ffmpeg command on machine level (not in the container), the command seems to run.

Any hint what I could do? Should I reinstall using bullseye 32bit, alternate firmware, alternate ffmpeg ?

### Version

0.11.0-be7b858 [v0.11-rc3]

### Frigate config file

```yaml
ffmpeg:
  global_args: -hide_banner -loglevel warning
#  hwaccel_args: []
  hwaccel_args: -c:v h264_v4l2m2m
  input_args: -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 
  output_args:
    detect: -f rawvideo -pix_fmt yuv420p
    record: -f segment -segment_time 60 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy -an
    rtmp: -c copy -f flv
```

### docker-compose file or Docker CLI command

```yaml
frigate:
    container_name: frigate
    restart: unless-stopped
    privileged: true
    image: blakeblackshear/frigate:0.11.0-rc3
    devices:
      - /dev/bus/usb:/dev/bus/usb
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /data/frigate/config-v11.yml:/config/config.yml:ro
      - /data/frigate/v11-media:/media/frigate
      - type: tmpfs
        target: /tmp/cache
        tmpfs:
          size: 1GB
    ports:
      - "5000:5000"
```

### Relevant log output

```shell
2022-09-19T10:24:17.437071291Z [2022-09-19 12:24:17] watchdog.cam1                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.448199164Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam1: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.449597754Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam1: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:17.580406608Z [2022-09-19 12:24:17] watchdog.cam2                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.590243975Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.591299704Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.594970332Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.596418511Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.597128583Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.597780882Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam2: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:17.696092838Z [2022-09-19 12:24:17] watchdog.cam3                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.707560185Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam3: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.711523974Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam3: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:17.776341374Z [2022-09-19 12:24:17] watchdog.cam4                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.792220399Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.795211396Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.799271437Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.802042230Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.804870630Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.806394858Z [2022-09-19 12:24:17] watchdog.cam6                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.809074733Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.810431752Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam4: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:17.823196067Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.825088691Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.826430378Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.828191717Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.829271759Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.830435905Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.831739095Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.860473575Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.861511380Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam6: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:17.912735089Z [2022-09-19 12:24:17] watchdog.cam5                  INFO    : FFmpeg didnt exit. Force killing...
2022-09-19T10:24:17.923871517Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam5: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.925078326Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam5: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.927116513Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam5: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.928440627Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam5: Unable to read frames from ffmpeg process.
2022-09-19T10:24:17.929602126Z [2022-09-19 12:24:17] frigate.video                  ERROR   : cam5: ffmpeg process is not running. exiting capture thread...
2022-09-19T10:24:21.465840392Z [2022-09-19 12:24:21] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
2022-09-19T10:24:21.466634326Z [2022-09-19 12:24:21] frigate.record                 ERROR   : could not convert string to float: ''
```

### FFprobe output from your camera

```shell
ffmpeg -hide_banner -loglevel debug -c:v h264_v4l2m2m -i rtsp://user:pass@x.x.y.z:80/1 -f rawvideo pipe: > /dev/null
Splitting the commandline.
Reading option '-hide_banner' ... matched as option 'hide_banner' (do not show program banner) with argument '1'.
Reading option '-loglevel' ... matched as option 'loglevel' (set logging level) with argument 'debug'.
Reading option '-c:v' ... matched as option 'c' (codec name) with argument 'h264_v4l2m2m'.
Reading option '-i' ... matched as input url with argument 'rtsp://user:pass@x.x.y.z:80/1'.
Reading option '-f' ... matched as option 'f' (force format) with argument 'rawvideo'.
Reading option 'pipe:' ... matched as output url.
Finished splitting the commandline.
Parsing a group of options: global .
Applying option hide_banner (do not show program banner) with argument 1.
Applying option loglevel (set logging level) with argument debug.
Successfully parsed a group of options.
Parsing a group of options: input url rtsp://user:pass@x.x.y.z:80/1.
Applying option c:v (codec name) with argument h264_v4l2m2m.
Successfully parsed a group of options.
Opening an input file: rtsp://user:pass@x.x.y.z:80/1.
[tcp @ 0x5592737360] No default whitelist set
[tcp @ 0x5592737360] Original list of addresses:
[tcp @ 0x5592737360] Address x.x.y.z port 80
[tcp @ 0x5592737360] Interleaved list of addresses:
[tcp @ 0x5592737360] Address x.x.y.z port 80
[tcp @ 0x5592737360] Starting connection attempt to x.x.y.z port 80
[tcp @ 0x5592737360] Successfully connected to x.x.y.z port 80
[rtsp @ 0x5592734880] SDP:
v=0
o=StreamingServer 3331435948 1116907222000 IN IP4 x.x.y.q
s=h264.mp4
c=IN IP4 0.0.0.0
t=0 0
a=control:*
m=video 0 RTP/AVP 96
a=control:trackID=0
a=rtpmap:96 H264/90000
a=fmtp:96 packetization-mode=1; sprop-parameter-sets=Z00AHpY1QWh7TcBAQECA,aO48gA==
a=videoinfo:720*480*20*4096
m=audio 0 RTP/AVP 0
a=control:trackID=1
a=rtpmap:0 PCMU/8000
a=ptime:20

[rtsp @ 0x5592734880] video codec set to: h264
[rtsp @ 0x5592734880] RTP Packetization Mode: 1
[rtsp @ 0x5592734880] Extradata set to 0x5592739120 (size: 27)
[rtsp @ 0x5592734880] audio codec set to: pcm_mulaw
[rtsp @ 0x5592734880] audio samplerate set to: 8000
[rtsp @ 0x5592734880] audio channels set to: 1
[rtp @ 0x559273ac20] No default whitelist set
[udp @ 0x559273af20] No default whitelist set
[udp @ 0x559273af20] end receive buffer size reported is 425984
[udp @ 0x559274b1f0] No default whitelist set
[udp @ 0x559274b1f0] end receive buffer size reported is 425984
[rtsp @ 0x5592734880] setting jitter buffer size to 500
[rtp @ 0x559275b870] No default whitelist set
[udp @ 0x559275bb10] No default whitelist set
[udp @ 0x559275bb10] end receive buffer size reported is 425984
[udp @ 0x559276bde0] No default whitelist set
[udp @ 0x559276bde0] end receive buffer size reported is 425984
[rtsp @ 0x5592734880] setting jitter buffer size to 500
[rtsp @ 0x5592734880] hello state=0
[h264 @ 0x5592738cd0] nal_unit_type: 7(SPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] nal_unit_type: 8(PPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] nal_unit_type: 7(SPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] nal_unit_type: 8(PPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] unknown SEI type 229
[h264 @ 0x5592738cd0] nal_unit_type: 7(SPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] nal_unit_type: 8(PPS), nal_ref_idc: 3
[h264 @ 0x5592738cd0] nal_unit_type: 6(SEI), nal_ref_idc: 0
[h264 @ 0x5592738cd0] nal_unit_type: 5(IDR), nal_ref_idc: 3
[h264 @ 0x5592738cd0] unknown SEI type 229
[h264 @ 0x5592738cd0] Format yuvj420p chosen by get_format().
[h264 @ 0x5592738cd0] Reinit context to 720x480, pix_fmt: yuvj420p
[h264 @ 0x5592738cd0] nal_unit_type: 1(Coded slice of a non-IDR picture), nal_ref_idc: 3
    Last message repeated 5 times
[h264 @ 0x5592738cd0] unknown SEI type 229
    Last message repeated 4 times
[rtsp @ 0x5592734880] max_analyze_duration 5000000 reached at 5000000 microseconds st:0
Input #0, rtsp, from 'rtsp://user:pass@x.x.y.z:80/1':
  Metadata:
    title           : h264.mp4
  Duration: N/A, start: 0.080000, bitrate: N/A
  Stream #0:0, 107, 1/90000: Video: h264 (Main), 1 reference frame, yuvj420p(pc, bt709, progressive, left), 720x480, 0/1, 25 tbr, 90k tbn
  Stream #0:1, 0, 1/8000: Audio: pcm_mulaw, 8000 Hz, mono, s16, 64 kb/s
Successfully opened the file.
Parsing a group of options: output url pipe:.
Applying option f (force format) with argument rawvideo.
Successfully parsed a group of options.
Opening an output file: pipe:.
[pipe @ 0x5592793bb0] Setting default whitelist 'crypto,data'
Successfully opened the file.
[h264_mp4toannexb @ 0x5592734760] The input looks like it is Annex B already
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video31
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-encode_image' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video23
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in unknown mode
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video22
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video21
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video20
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video19
[h264_v4l2m2m @ 0x55929318c0] driver 'rpivid' on card 'rpivid' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video18
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-image_fx' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video16
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in unknown mode
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video15
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video14
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video13
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-isp' on card 'bcm2835-isp' in splane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video12
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-isp' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video11
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-encode' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] v4l2 output format not supported
[h264_v4l2m2m @ 0x55929318c0] probing device /dev/video10
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-decode' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] Using device /dev/video10
[h264_v4l2m2m @ 0x55929318c0] driver 'bcm2835-codec' on card 'bcm2835-codec-decode' in mplane mode
[h264_v4l2m2m @ 0x55929318c0] requesting formats: output=H264 capture=YU12
[h264_v4l2m2m @ 0x55929318c0] output: H264 16 buffers initialized: 0720x0480, sizeimage 00524288, bytesperline 00000000
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (h264_v4l2m2m) -> rawvideo (native))
Press [q] to stop, [?] for help
cur_dts is invalid st:0 (0) [init:0 i_done:0 finish:0] (this is harmless if it occurs once at the start per stream)
[h264_v4l2m2m @ 0x55929318c0] capture: YU12 20 buffers initialized: 0720x0480, sizeimage 00529920, bytesperline 00000736
cur_dts is invalid st:0 (0) [init:0 i_done:0 finish:0] (this is harmless if it occurs once at the start per stream)

<here it hung>

^C^C^CReceived > 3 system signals, hard exiting
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

no name

**Comment by NickM-27** [comment]
Please provide your entire frigate config

If you disable hwaccel for the dahua cameras does it work for the others?

**Comment by S-Przybylski** [comment]
Quick checked with only one active "no name cam" with hardware acceleration enabled - green display. If I turn of the hardware accleration in the ffmpeg setting, the "no name" cam is working... The dahua cameras do not affect this...
For me this seems to be related somehow to the ffmpeg and maybe firmware version (I already had in buster #3332)

Additional details of the config as requested:
````
...
detectors:
  coral:
    type: edgetpu
    device: usb
...
record:
  enabled: False
  expire_interval: 60
  retain:
    days: 0
    mode: all
  events:
    pre_capture: 5
    post_capture: 5
    objects:
      - person
    required_zones: []
    retain:
      default: 30
      objects:
        person: 90

snapshots:
  enabled: False
  clean_copy: True
  timestamp: False
  bounding_box: False
  crop: False
  height: 175
  quality: 70
  required_zones: []
  retain:
    default: 30
    objects:
      person: 90

ffmpeg:
  global_args: -hide_banner -loglevel warning
#  hwaccel_args: []
  hwaccel_args: -c:v h264_v4l2m2m
  input_args: -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 
  output_args:
    detect: -f rawvideo -pix_fmt yuv420p
    record: -f segment -segment_time 60 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy -an
    rtmp: -c copy -f flv

...
  cam:
    ffmpeg:
      inputs: 
        - path: rtsp://camerauser:{FRIGATE_RTSP_PASSWORD}@x.x.y.z:80/1 #no name camera
#        - path: rtsp://camerauser:{FRIGATE_RTSP_PASSWORD}@x.x.y.z:554/cam/realmonitor?channel=1&subtype=1 #dahua hdb3841e

          roles:
            - detect
            - record
    detect:
      width: 720 #no name
      height: 480 #no name
#      width: 704 #dahua
#      height: 576 #dahua
      fps: 5
      enabled: True
      max_disappeared: 20
    motion:
      mask: [not shown]
    best_image_timeout: 10
    record:
      enabled: True
      expire_interval: 60
      retain:
        days: 0
        mode: all
      events:
        pre_capture: 5
        post_capture: 5
        objects:
          - person
        required_zones: []
        retain:
          default: 30
          objects:
            person: 90  
    rtmp:
      enabled: False
    live:
      height: 480
      quality: 8
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: False
      crop: False
      height: 175
      required_zones: []
      retain:
        default: 30
        objects:
          person: 90
    mqtt:
      enabled: True
      timestamp: True
      bounding_box: True
      crop: True
      height: 270
      quality: 70
      required_zones: []
    objects:
      track:
        - person
      filters:
        person:
          mask:
            - [not shown]
    timestamp_style:
      position: "tl"
      format: "%Y-%m-%d %H:%M:%S"
````

**Comment by NickM-27** [comment]
Interesting, unfortunately the logs don't give much info as to what's wrong. 

If it works on your host then might be worth trying https://deploy-preview-2829--frigate-docs.netlify.app/configuration/advanced#custom-ffmpeg-build

**Comment by S-Przybylski** [comment]
Is it possible to map the installed raspi ffmpeg version into the container? Simply map /usr:/usr/lib/btbn-ffmpeg:ro doesn't work.

**Comment by NickM-27** [comment]
Okay well it's good to know that the BtBn is not working so there's not much we can currently do. 

Maybe try mapping. `/usr/bin/ffmpeg:/usr/lib/btbn-ffmpeg/bin/ffmpeg:ro` and see what happens. What error did you get when trying before with /usr ?

**Comment by S-Przybylski** [comment]
The following ffmpeg logs include the last 100 lines prior to exit.
2022-09-19T16:45:53.628048380Z [2022-09-19 18:45:53] ffmpeg.cam4.detect             ERROR   : ffmpeg: error while loading shared libraries: libavdevice.so.58: cannot open shared object file: No such file or directory

**Comment by NickM-27** [comment]
I see, yeah that may not be easily doable.

The only other thing I can suggest is maybe go back to the camera and see if you can adjust any of the settings that might help.

I'm not really sure there's much we can do. We are tracking a few different bugs with ffmpeg that have popped up in more recent versions of the RPi firmware and other packages. Especially as it affects some types of cameras and not others.

**Comment by S-Przybylski** [comment]
I have reinstalled Frigate on a Asus PN41 Celeron N4500 (with USB Coral). Works great. RASPI 4 is no longer in use for Frigate.

**Comment by jokerigno** [comment]
HI @S-Przybylski sorry for necroposting but I was wondering if you can share your HW Accel settings for ASUS PN41 with N4500. I'm usingi frigate inside HA and have some performance issues (VM locks and have to reboot from time to time).

Thank you in advance!

**Comment by S-Przybylski** [comment]
Dear @jokerigno 
I have attached 9 cam's to the PN41 (Ubuntu, 16 GByte RAM, Coral enabled, docker, docker-compose installed). Each CPU has a load round about of 65% permanently. 
Up to now, I never had an issue that the PN41 or the container hung.
But yes from time to time I also need to restart Frigate (I have this recently noticed this too), because sometimes the detection hang somehow or my HA dashboard show massively delayed streams.
I also saw that the container utilized much memory (>4GByte in total). It seems that at least an internal component of v0.11 seems to have memory issues.

My current solution: Manually restart the container using the ha-dockermon container (image: philhawthorne/ha-dockermon) inside HomeAssistant. I use this tool, because Frigate is running on a dedicated system.
